nntrainer/utils/util_func.cpp

```diff
@@ -49,7 +49,8 @@ float exp_util(float x) { return exp(x); }
 
 // This is 2D zero pad
 // TODO : Optimize for multi dimention padding
-Tensor zero_pad(int batch, Tensor const &in, unsigned int const *padding) {
+Tensor zero_pad(int batch, Tensor const &in, unsigned int const *padding,
```

Comment:
This patch does not use the output. How about we return void here to reduce additional copy?

Reply:
done